README.md

```diff
@@ -160,6 +160,7 @@ Client Instrumentation|Every Tuesday at 9:00 AM PT (primary) and every Monday at
 Kubernetes Operator|Every other week on Thursday at 09:00 PT|[Google Doc](https://docs.google.com/document/d/1Unbs2qp_j5kp8FfL_lRH-ld7i5EOQpsq0I4djkOOSL4)|[Slack](https://cloud-native.slack.com/archives/C033BJ8BASU)|
 Community Demo Application|Every Monday at 8:15 PT|[Google Doc](https://docs.google.com/document/d/16f-JOjKzLgWxULRxY8TmpM_FjlI1sthvKurnqFz9x98/edit)|[Slack](https://cloud-native.slack.com/archives/C03B4CWV4DA)|
 Functions as a Service (FAAS) |Every Tuesday at 12:00 PM PT |[Google Doc](https://docs.google.com/document/d/187XYoQcXQ9JxS_5v2wvZ0NEysaJ02xoOYNXj08pT0zc/edit#)|[Slack](https://cloud-native.slack.com/archives/C04HVBETC9Z)|
+OpenTelemetry Profilng WG | Every other week on Thursday at 08:00 AM PT |[Google Doc](https://docs.google.com/document/d/1D-1MbT3MWpKM61JgVFx_TQi4vqWLLIZp6biK5MoHafI/edit?usp=sharing)|[Slack](https://cloud-native.slack.com/archives/C03J794L0BV)|
```

Comment:
@Rperry2174 can you transfer this doc to the OpenTelemetry Governance google account?

from #1147:

> ## Process to transfer ownership
> 
> 1. Click the share button 
> 2. Invite cncf-opentelemetry-governance at lists.cncf.io to be an editor
> 3. Click the share button again
> 4. Click on the arrow next to the governance committee email and select "Transfer Ownership" 
> 5. Click "Send Invitation"

Reply:
this look correct? 
<img width="683" alt="image" src="https://github.com/open-telemetry/community/assets/23323466/17f019db-daae-45dc-a98c-2c1c1043844e">

Reply:
yup!

Reply:
done!